db/migrations/public/20231127222742_create-purposes-view.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const viewName = 'purposes'
+
+exports.up = function (knex) {
+  return knex
+    .schema
+    .createView(viewName, (view) => {
+      // NOTE: We have commented out unused columns from the source table
+      view.as(knex('purposes_uses').withSchema('water').select([
+        'purpose_use_id AS id',
+        'legacy_id',
+        'description',
+        'loss_factor',
+        'is_two_part_tariff',

```

Comment:
Same as previous comment

```suggestion
        'is_two_part_tariff AS two_part_tariff',

```